internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -660,8 +657,30 @@ private ConfigKeys() {
    */
   public static final String SERVER_FORKED_PROCESS_JVM_ARGUMENT_LIST = "server.forked.process.jvm.arg.list";
 
+  /**
+   * Timeout for connection between main process and forked ingestion process. If heartbeat is not refreshed within this
+   * timeout, both processes should act to reconstruct the state in order to restore connection and service.
+   */
+  public static final String SERVER_INGESTION_ISOLATION_CONNECTION_TIMEOUT_MS =
+      "server.ingestion.isolation.connection.timeout.ms";
+
+  /**
+   * Timeout for single ingestion command request sent from main process to forked ingestion process.
+   */
   public static final String SERVER_INGESTION_ISOLATION_REQUEST_TIMEOUT_SECONDS =
-      "server.ingestion.isolation.request.timout.seconds";
+      "server.ingestion.isolation.request.timeout.seconds";
```

Comment:
It seems confusing to have some timeouts in seconds and others in ms... can we align those configs on the same unit?

Reply:
Okay, I will try to change them into seconds. MS does not make much sense here.